### PR TITLE
Bug 1456549 - Fix search rollup timeout and increase instance count

### PR DESCRIPTION
**Pull request by acmiyaguchi**
[Bug Link](https://bugzilla.mozilla.org/show_bug.cgi?id=1456549)

The search rollup job has been failing intermittently over the last few weeks. I left out some key arguments in the DAG definition that prevented the intended email recipients from getting the retry alerts. I've also bumped up the instance count and timeout value.

### CODE REVIEW

**Comment by acmiyaguchi** [comment]
@sunahsuh Bumping for review, also feel free to untag yourself.

I verified that there was an issue with the days that timed out related to a OOM error. Bumping the instances up to 3 fixes the underlying problem.

File: dags/search_rollup.py [hunk 1]
```diff
@@ -30,11 +29,17 @@
...
+
+    This can be called with an optional DAG passed into `upstream`. The rollup
+    job will inherit the default values of the referenced DAG.
+    """
 
     search_rollup = EMRSparkOperator(
         task_id="search_rollup_{}".format(mode),
         job_name="{} search rollup".format(mode).title(),
+        owner="amiyaguchi@mozilla.com",
+        email=default_args["email"],
```
Comment (sunahsuh) [inline dags/search_rollup.py]: So, wait, default_args doesn't apply the email default arg? 😧

File: dags/search_rollup.py [hunk 2]
```diff
@@ -30,11 +29,17 @@
 
 
 def add_search_rollup(dag, mode, instance_count, upstream=None):
-    """Create a search rollup for a particular date date"""
+    """Create a search rollup for a particular date date
+
+    This can be called with an optional DAG passed into `upstream`. The rollup
```
Comment (sunahsuh) [inline dags/search_rollup.py]: I think you mean "task" instead of "DAG" here

File: dags/search_rollup.py (hunk 1 above)
Comment (sunahsuh) [reply dags/search_rollup.py]: Oh wait, I see.. you're referencing the `default_args` variable in the file. Hmm, I think in an ideal world we'd pass in those parameters to the `add_search_rollup` function, or pass in `default_args` with the double-star

File: dags/search_rollup.py [hunk 3]
```diff
@@ -15,12 +15,11 @@
         'telemetry-alerts@mozilla.com',
         'amiyaguchi@mozilla.com',
         'harterrt@mozilla.com',
-        'spenrose@mozilla.com'
     ],
     'email_on_failure': True,
     'email_on_retry': True,
     'retries': 2,
-    'retry_delay': timedelta(minutes=30),
+    'retry_delay': timedelta(hours=2),
```
Comment (sunahsuh) [inline dags/search_rollup.py]: Is this something that's relevant just for this DAG's task instance and not for the task in main summary?

Comment (acmiyaguchi) [reply dags/search_rollup.py]: Actually, this doesn't do what I meant it to do. This should be omitted.

**Comment by acmiyaguchi** [comment]
@sunahsuh I moved the function to the top for clarity and removed some unnecessary defaults. Hopefully it should read a little clearer now.

**Pull request merged**